### PR TITLE
add README

### LINKED ISSUE #7: Add Readme
The repo should have a readme! :)

### PR DESCRIPTION
**Pull request by robertjallen**
# Description
Fixes # (issue # from issue tracker)

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status
- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ X] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?
- [ ] Unit Tests
- [ ] Integration Tests
- [ ] User Tests

# Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ X] My changes generate no new warnings
- [X ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X ] There are no merge conflicts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# UnderDog Dev's:
```
Comment (buggtb) [inline README.md]: Don't believe there's an apostrophe in Devs because there wouldn't be one in Developers

**Review by buggtb — COMMENTED** [review]
Outside of the apostrophe, looks good.

File: README.md (hunk 1 above)
Comment (robertjallen) [reply README.md]: fixed

Comment (robertjallen) [reply README.md]: ready to be merged

**Review by buggtb — APPROVED** [review]

**Comment by buggtb** [comment]
LGTM! Merge away.

**Pull request merged**